core-bundle/src/EventListener/DataContainer/DefaultOperationsListener.php

```diff
@@ -96,12 +100,17 @@ private function getDefaults(string $table): array
         }
 
         if ($ctable) {
-            $operations += [
-                'children' => [
-                    'href' => 'table='.$ctable,
-                    'icon' => 'children.svg',
-                ],
-            ];
+            $this->framework->getAdapter(Controller::class)->loadDataContainer($ctable);
+
+            if (DataContainer::MODE_TREE_EXTENDED !== ($GLOBALS['TL_DCA'][$ctable]['list']['sorting']['mode'] ?? null)) {
```

Comment:
Why this?

Reply:
because articles (extended tree) are a child table of pages (regular tree), but the page list does not/should not have a children operation.